.eslintrc.json

```diff
@@ -17,6 +17,7 @@
   },
   "rules": {
     "no-bitwise": "off",
+    "no-plusplus": "off",
```

Comment:
why is this turned off? use i += 1; instead

Reply:
The explanation on why we should not use ++ is, that the code gets "cryptic". But as i always use it on a single line, it should not lead to errors or complicated code. Excerpt from [stack overflow](https://stackoverflow.com/questions/3000276/the-unexpected-error-in-jslint). Also the IDE generates loops with ++ so should we still go with += 1 ?

Reply:
refer to https://eslint.org/docs/rules/no-plusplus, it makes sense to not use the operator